wpiunits/src/test/java/edu/wpi/first/units/MeasureTest.java

```diff
@@ -196,6 +196,35 @@ void testDivideMeasure() {
     assertEquals(result.unit(), Units.Value);
   }
 
+  @Test
+  void testInverse() {
+    // Time
+    var m1 = Units.Seconds.of(2);
+    var result = m1.inverse();
+    assertEquals(0.5, result.magnitude());
+    assertEquals(Units.Hertz, result.unit());
```

Comment:
This test fails because `Measure.divide` does not return velocity units when dividing by time. Looks like #6696 will add that behavior